airbyte-integrations/connectors/source-zendesk-support/source_zendesk_support/schemas/tickets.json

```diff
@@ -179,6 +179,26 @@
     },
     "forum_topic_id": {
       "type": ["null", "integer"]
+    },
+    "custom_status_id": {
+      "type": ["null", "integer"]
+    },
+    "fields": {
+      "type": ["null", "array"],
+      "items": {
+          "type": ["null", "object"],
+          "properties": {
+              "id": {
+                "type": ["null", "integer"]
+              },
+              "value": {
+                "type": ["null", "boolean", "string"]
+              }
+          }
+      }
```

Comment:
@asafepy this part of the code. This key called `fields` I don't see in ticket API documentation

Reply:
yes, you is correct...  but in my request this field comes in the payload...

Reply:
I'm going to change here and remove this part of the scheme

Reply:
@marcosmarxm i`m remove this field from schema.